frontend/.env

```diff
@@ -0,0 +1 @@
+VITE_API_URL=http://localhost:8080/
```

Comment:
I can do that, but will everyone else remember that this is needed when creating docker images?

Reply:
It isn't. This parameter can be passed as a normal env var. Another thing is that the error says explicitly (or if not, it should) that such an environment variable is missing. If it is not sensitive data then it can be added to env_temp for easier copying, but we should definitely not add this file to the repo.

Reply:
Ok, I will delete .env but leave .env.production (needed for pipeline)